validation/citrus-validation-json/src/test/java/org/citrusframework/validation/json/JsonElementValidatorTest.java

```diff
@@ -72,6 +71,14 @@ public static JsonAssertion[] validJsonPairsIfNotStrict() {
                 new JsonAssertion(
```

Comment:
I am pretty sure, that this would have failed in the past.

Reply:
because it did only take suffixes into account, but prefixes were "strict"?

Reply:
@bbortt yes exactly - I'll do my best to document that a bit more concise.